jquery-sortable-lists.js

```diff
@@ -129,8 +131,10 @@
 
 		if ( setting.opener.active )
 		{
-			if ( ! setting.opener.open ) throw 'Url for opener.open image is not defined';
-			if ( ! setting.opener.close ) throw 'Url for opener.close image is not defined';
+			if ( ! setting.opener.as || (setting.opener.as !== 'img' && setting.opener.as !== 'html') )
```

Comment:
This should be enough 
setting.opener.as !== 'img' && setting.opener.as !== 'html'
soholdn't be?